discord-bot/bot/extensions/example.py

```diff
@@ -0,0 +1,406 @@
+# -*- coding: utf-8 -*-
+"""Example plugins for reference.
+
```

Comment:
why do we need this 406 lines example file in the project? Could you maybe link this in a readme?